src/pages/About/styleds.tsx

```diff
@@ -227,7 +227,7 @@ export const Powered = styled.div`
   justify-content: center;
   gap: 52px;
   & > * {
-    width: calc(100% / 6 - 52px);
+    width: calc(100% / 5 - 52px);
```

Comment:
can we update it to dynamic?

Reply:
wdym

Reply:
100% / xxx.length - 52px

Reply:
I afraid not. This number is number of columns, it has nothing to do with the list length. We have to set it manually to ensure the UI not breaking.